### PR TITLE
Add script to generate issue for status of testing of the rc

### PR DESCRIPTION
**Pull request by potiuk**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/main/UPDATING.md).

### CODE REVIEW

File: dev/ISSUE_TEMPLATE.md.jinja2 [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+
```
Comment (uranusjr) [inline dev/ISSUE_TEMPLATE.md.jinja2:2]: Why the blank lines? (Doesn't really matter but I'm curious.)

Comment (potiuk) [reply dev/ISSUE_TEMPLATE.md.jinja2:2]: I like whitespace :D

Comment (potiuk) [reply dev/ISSUE_TEMPLATE.md.jinja2:2]: Hnestly - it reads better when they are separated - .md renders them as separate paragraphs and by separating them with spaces it "looks" similar also pre-rendered. I think I do it subconsciously, not even thinking about it.

Comment (potiuk) [reply dev/ISSUE_TEMPLATE.md.jinja2:2]: I also like to keep the paragrpaphs "justified" in the .md - even though .md will nicely wrap them even if one paragraph is long line, it makes it difficult to read in IDE's without soft-wrapping the lines. I think this is alll "let-it-look as close to rendered-version in sources" kind of subconscious thinking.

Comment (potiuk) [reply dev/ISSUE_TEMPLATE.md.jinja2:2]: Actually your question @uranusjr got me thinking :). 

I did notice that I prefer more whitespace (particularly empty lines) than others. When you look at breeze code (especially output it produces) it was usually a fight between me and @mik-laj on adding/removing empty lines here and there 🗡️ 

I thought a bit about it and I think it came with time. I also used to configure my IDEs, fonts etc. to fit as much as possible on one screen for years, but I've been working for years in Mobile App Development area and we went there through "Material Design" revolution. Where designers taught us that whitespace plays a big role in visual communication - almost as big as content. Adding more space to separate different "topics " allow to focus only on one of those at a time much better mentally.

I even tend to play with Material Design theme in IntelliJ from time to time, but it's simply a bit too slow with all the bells and whistles it comes with and I tend to switch back (but it's back-forth more often than not) . On my Home PC which is super-fast I turn it on a bit more often, but on the Chromebook I use when travelling (like now :) it visibly lags. So for me adding more whitespace is bringing a bit more of the "Material Design" thinking without slowing things down I think.

At least that's my hypothesis why I am doing it - it's really subconscious to be honest.

Comment (potiuk) [reply dev/ISSUE_TEMPLATE.md.jinja2:2]: And that would actually explains why it's at the top :D. Simply getting some space between the top of the window and first "significant" word there makes it more visually appealing 🎨

**Review by ashb — COMMENTED** [review]
I have no problem with the code, but I am not sure that the approach works for the scale of some of our releases. For instance 2.2.0 had 500 commits!

**Comment by potiuk** [comment]
> I have no problem with the code, but I am not sure that the approach works for the scale of some of our releases. For instance 2.2.0 had 500 commits!

Quite agree. I initially proposed to use it mostly for bugfix releases. There are likely ways we could make it work for major releases as well - but the current implementation only works for bugfix releases.

For Major releases we could only extract bugfixes + (maybe) high level new features (but for that towncrier implementation or similar would be necessary). Also we could automatically exclude bugfixes cherry-picked to other releases.

I think it's a good base to experiment when we get to 2.3.0 - then we could see how many "new" things are left after such extraction and see if it is manageable.

File: dev/README_RELEASE_AIRFLOW.md [hunk 2]
```diff
@@ -246,6 +247,27 @@ Production Docker images should be manually prepared and pushed by the release m
 This will wipe Breeze cache and docker-context-files in order to make sure the build is "clean". It
 also performs image verification before pushing the images.
 
+## Prepeare issue for testing status of rc
```
Comment (potiuk) [inline dev/README_RELEASE_AIRFLOW.md]:
```suggestion
## Prepeare issue for testing status of rc

For now this part works for bugfix releases only, for major releases we will experiment in the future and
see if there is a way to only extract those bugfixes and high-level changes to make the list manageable.

```

Comment (ashb) [reply dev/README_RELEASE_AIRFLOW.md]: You mean "minor" or "feature" -- Major is 3.0.0

**Comment by potiuk** [comment]
> For minor/feature (x.y.0) releases we could only extract bugfixes + (maybe) high level new features (but for that towncrier implementation or similar would be necessary). Also we could automatically exclude bugfixes cherry-picked to other releases.

I updated the docs accordingly.

File: dev/README_RELEASE_AIRFLOW.md (hunk 2 above)
Comment (potiuk) [reply dev/README_RELEASE_AIRFLOW.md]: true

**Comment by potiuk** [comment]
Should be good to merge I guess?

**Comment by potiuk** [comment]
Tested the process and made a few small fixes with 2.2.2rc2: https://github.com/apache/airflow/issues/19515 - I had to manually remove some issues (doc/static checks)- but that was painless. I think it should be ready to merge.

**Review by uranusjr — APPROVED** [review]
Browsed through the code and did not see anything wrong. Given that this works in practice it should be ready.

**Pull request merged**